src/components/Maps/MapPage.tsx

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import { useQuery } from 'react-apollo';
+import GoogleMapReact from 'google-map-react';
+
+import MenuAppBar from '../common/MenuAppBar';
+import InfoWindow from './InfoWindow';
+import { LocationPin, GET_PINS_BY_EVENT_ID } from '../../graphql/queries/maps';
+
+const MapPage = ({
+  match: {
+    params: { eventId },
+  },
+}: {
+  match: { params: { eventId: string } };
+}) => {
+  const defaultMap = {
+    center: { lat: 43.470846, lng: -80.538473 },
```

Comment:
this will probably have to be tweaked and not hardcoded - potentially the coordinates of the `event` which the map can centre around?

Reply:
Yep exactly.